pkg/operation/botanist/component/dependencywatchdog/bootstrap.go

```diff
@@ -413,27 +433,38 @@ func (b *bootstrapper) podLabels() map[string]string {
 	return nil
 }
 
-func (b *bootstrapper) containerCommand() []string {
+func (b *bootstrapper) containerArgs() []string {
```

Comment:
`containerCommand` was actually correct. if you see the pod specification you will usually put the command along with its arguments in `spec.containers[<index>].command`